src/main/java/org/rumbledb/runtime/functions/input/UnparsedTextLinesFunctionIterator.java

```diff
@@ -65,7 +65,7 @@ public UnparsedTextLinesFunctionIterator(
         if (this.children.size() == 1) {
             strings = SparkSessionManager.getInstance()
                 .getJavaSparkContext()
-                .textFile(uri.toString());
+                .textFile(uri.toString(), 10);
```

Comment:
Can you extract this minPartitions number to a Constant

Reply:
Thank you for taking care of it, Can!